SUP_Board_JSON.txt

```diff
@@ -0,0 +1,9 @@
+{
+  "brand": "Decathlon",
+  "model": "Hurley",
+  "size": "335X85X15",
+  "color": "Mixed Color",
+  "maximum load": "150kg",
+  "weight of the packaging" : "13kg",
```

Comment:
Here suggestion is to avoid spaces in property names as it can make further processing harder

not good: `weight of the packaging`
good: `weight_of_the_packaging`
better: `packaging_weight_kg` 

Also , it always makes sense to keep numeric values as integer, and describe the measurement unit in the property itself, as integers associated with values are always easier to process

```json
...
"maximum_load_kg": 150,
"weight_of_the_packaging_kg" : 13,
 ...
```